pkg/volume/azure_file/azure_file.go

```diff
@@ -104,6 +105,13 @@ func (plugin *azureFilePlugin) newMounterInternal(spec *volume.Spec, pod *v1.Pod
 	if err != nil {
 		return nil, err
 	}
+	nameSpace := pod.Namespace
+	if spec.PersistentVolume != nil {
+		ns, err := getSecretNamespace(plugin.host.GetKubeClient(), spec.PersistentVolume)
```

Comment:
Mounter has to work with new provisioner to pick up custom namespace. Existing mounter won't be able to look up secret in other namespace.